shukla/Dockerfile

```
@@ -0,0 +1,9 @@
+FROM ubuntu
```

```suggestion
FROM ubuntu:22.04
```

While this is valid, this is subject to breakage as you haven't specified a tag of the `ubuntu` image.